lib/navigation/benchmark/benchmark.cpp

```diff
@@ -0,0 +1,98 @@
+#include "benchmark.hpp"
+
+#include <core/timer.hpp>
+
+namespace hyped::navigation::benchmark {
+
+TrajectoryError::TrajectoryError(const core::Trajectory &expected, const core::Trajectory &actual)
+    : displacement(std::abs(expected.displacement - actual.displacement)),
+      velocity(std::abs(expected.velocity - actual.velocity)),
+      acceleration(std::abs(expected.acceleration - actual.acceleration))
+{
+}
+
+Benchmark::Benchmark(const core::ITimeSource &time_source,
```

Comment:
Think the format of raw data will be changing soon to a format which includes timestamps? @SnickeyX?

Reply:
#31 is adding an acceleromter struct so `rawimudata` isnt an array of array of acceleration with the axis as its index but instead an array of an acceleration struct with acceleration across the three axes + timestamp from when we got the data (though timstamp in that PR rn needs to not call chrono directly, gonna review in a bit)